frontend/src/index.css

```diff
@@ -13,82 +11,67 @@ body {
     -webkit-font-smoothing: antialiased;
     -moz-osx-font-smoothing: grayscale;
     background-color: white;
+    
+}
+
+.row{
+    margin-right:0rem;
 }
 
 .input-group-text {
-    background-color: white;
+    background-color: #F5F5F5;
     border-radius: 20rem;
     color: #495057bf;
+    padding: 0rem .75rem;
 }
 
 .form-control {
     border-radius: 20rem;
+    background-color: #F5F5F5;
+    font-size:0.96rem;
 }
 
-.todo-title {
-    cursor: pointer;
-}
-
-.tab-list>span {
-    padding: 5px 8px;
-    border: 1px solid #282c34;
-    border-radius: 10px;
-    margin-right: 5px;
-    cursor: pointer;
-}
-
-.tab-list>span.active {
-    background-color: #282c34;
-    color: #ffffff;
-}
-
-.btn-light {
-    background-color: #e0e0e075;
-    margin-bottom: 7px;
-    border-radius: 0px;
-    padding-top: 10px;
-    padding-bottom: 10px;
-    box-shadow: 1px 1px #80808000;
-}
-
-.row {
-    margin-right: 0px;
-    margin-left: 0px;
-    height: 100vh;
+.title{
+    font-weight: bold;
 }
 
-.h3 {
-    font-family: 'Libre Baskerville', serif;
+.clientadvisorBackground{
+    background-color: #CCCCCC;
 }
 
-.p-4 {
-    padding: 5rem!important;
+.loginBackground{
+    background-color: white;
 }
 
-.btn.blue-purple-gradient {
-    background: linear-gradient(45deg, rgb(114, 30, 190), rgb(50, 96, 179));
-    border-radius: 5rem;
+.btn {
+    margin: 0.375rem;
+    color: white;
+    text-transform: uppercase;
+    word-wrap: break-word;
+    white-space: normal;
+    cursor: pointer;
+    border: 0;
+    padding: 0.84rem 2.14rem;
+    font-size: 0.81rem;
 }
-
-.blue-purple-gradient {
-    background: linear-gradient(45deg, rgb(114, 30, 190), rgb(50, 96, 179));
-    border-radius: 0rem 5rem 0rem 0rem;
-    margin-left: -15px;
-    margin-top: 20px;
+.btn:hover {
+    color: white;
+    box-shadow: 0rem 0.125rem 0.3125rem 0.3125rem rgba(0, 0, 0, 0.16), 0rem 0.125rem 0.625rem 0rem rgba(0, 0, 0, 0.12);
+    transition: color 0.15s ease-in-out, background-color 0.15s ease-in-out, border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
 }
 
-.fontawesomeicon.blue-purple-gradient {
-    color: linear-gradient(45deg, rgb(114, 30, 190), rgb(50, 96, 179));
+.spacerPrimary{
```

Comment:
lets try to make all our css class names hyphenated and not camelCase (i see a few diff ones in here so lets just change them all earlier than later)